### PR TITLE
v1.3 Enhance find command

### LINKED ISSUE #60: v1.3 Feature: Extend the find command to groups, persons and events
### User stories
- As a user, I can find items in FumbleLog with a find_all command so that I can find persons and events together
- As a user, I can find persons in FumbleLog with a find_person command so that I can just find persons without affecting the events.
- As a user, I can find events in FumbleLog with a find_person command so that I can just find events without affecting the persons.

### Details
- find_all command that will search through groups, persons, and events by keyword
  - Format: find_all KEYWORD [KEYWORDS]
- find_person command that will search through names and groups of a person by keyword
  - Format: find_person KEYWORD [KEYWORDS]
- find_event command that will search through names and groups of an event by keyword
  - Format: find_event KEYWORD [KEYWORDS]

### PR DESCRIPTION
**Pull request by Ken-Lai**
- Enhance `find` command to `find_all` command
  - Able to find keywords in persons and events together
  - Format: `find_all KEYWORD [KEYWORD]`
- Add `find_person` and `find_event` command
  - Finds keywords in person and event separately
  - Format: `find_person/find_event KEYWORD [KEYWORD]`
- Add test cases for all commands mentioned above

### CODE REVIEW

**Review by Kurtyjlee — APPROVED** [review]
LGTM

File: src/main/java/seedu/address/logic/commands/FindAllCommand.java [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+/**
+ * Finds and lists all persons and events in address book whose name or group contains any of the argument keywords.
+ * Keyword matching is case insensitive.
+ */
+public class FindAllCommand extends Command {
+
+    public static final String COMMAND_WORD = "find_all";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all items whose names or groups contain any "
```
Comment (nicleongyj) [inline src/main/java/seedu/address/logic/commands/FindAllCommand.java:19]: maybe can change items to events or persons

**Review by nicleongyj — APPROVED** [review]
LGTM!

**Comment by leezhengjing** [comment]
LGTM

**Pull request merged**